### ISSUE TITLE
Any typescript definition for this?

### ISSUE THREAD

**Issue by Nomenator** [dgrubelic/vue-authenticate#117]
When trying to import this into boot.ts, I got the missing type definition error.

Been reading up on the TS documentation today, it looks like I will need to study the library itself extensively and understand all its ins and outs. Is there a definition for that available?

**Comment by tahq69** [comment]
+ 1 
Typescript definitions required to use with VueCLI 3 typescript project.

**Comment by leozero** [comment]
+1 It will be great to have this

**Comment by jalcine** [comment]
Yeah, lack of types are preventing from taking this to production.

**Comment by jalcine** [comment]
However, y'all can pass in "allowJs: true" if you just need it in a pinch.
